src/asset/config.ts

```diff
@@ -12,3 +12,43 @@ export const defaultConfig: Static<typeof AssetConfig> = {
 	accito: 60, // Accelerometer inactivity timeout in s: Hysteresis timeout for stillness detection.
 	nod: [] as DataModules[],
 } as const
+
+export const presetConfigs: Record<
+	string,
+	{
+		config: Static<typeof AssetConfig>
+		label: string
+		description: string
+	}
+> = {
+	parcel: {
+		config: {
+			act: false, // passive mode
+			mvres: 3600, // movement resolution
+			accath: 10, // Accelerometer activity threshold
+			accith: 5, // Accelerometer inactivity threshold
+			accito: 1200, // Accelerometer inactivity timeout
+			mvt: 21600, // Movement Timeout
+			actwt: 10,
+			gnsst: 10,
+			nod: [],
+		},
+		label: 'Parcel',
+		description: 'Used for tracking parcels.',
```

Comment:
Add more extensive description (explain what values are specifically set and why).